torcrawl.py

```diff
@@ -274,7 +274,7 @@ def main():
         lst = crawler.crawl()
         with open(out_path + "/links.txt", "w+", encoding="UTF-8") as file:
             for item in lst:
-                file.write(f"{item}\n")
+                file.write(f"{item} \n")
```

Comment:
Is this change necessary? From my point of view this extra white-space is not required.

Reply:
Oh I did it for making the manual search in links.txt easier. So that I can analyze and verify results. You can remove the extra whitespace.